### DISCUSSION TITLE
how can i build dynamic packge in arm64-linux

### DISCUSSION THREAD

**Discussion by sbjw** [microsoft/vcpkg#34024] [Q&A]
![image](https://github.com/microsoft/vcpkg/assets/19704945/ef47ab96-b061-4db7-8dd9-fca64245b742)

i wan't build gdal libary in arm64 linux, but i can't find arm64-linux-dynamic triplet in all triplets.

**Comment by Osyotr** [comment]
Create your own triplet: https://learn.microsoft.com/en-us/vcpkg/users/examples/overlay-triplets-linux-dynamic

**Comment by sbjw** [comment]
Thanks
